Source/Helper.swift

```diff
@@ -16,13 +18,89 @@ struct Helper {
     }
   }
 
-  static func videoOrientation() -> AVCaptureVideoOrientation {
-    switch UIDevice.current.orientation {
-    case .portrait: return .portrait
-    case .landscapeLeft: return .landscapeRight
-    case .landscapeRight: return .landscapeLeft
-    case .portraitUpsideDown: return .portraitUpsideDown
-    default: return .portrait
+  static func rotationTransform(_ configuration: Configuration) -> CGAffineTransform {
+    let currentOrientation = UIDevice.current.orientation
+    var result: CGAffineTransform
+
+    switch configuration.lockedOrientation {
```

Comment:
Couldn't this be simplified by simply checking if the current orientation is included in the allowed orientations mask?